app/src/ui/history/merge-call-to-action-with-conflicts.tsx

```diff
@@ -270,6 +270,7 @@ export class MergeCallToActionWithConflicts extends React.Component<
           <strong>{`${this.commitCount} ${pluralized}`}</strong>
           {` on top of `}
           <strong>{branch.name}</strong>
+          {`.`}
```

Comment:
I'd say it's better if we stick to the original style without `.` at the end of all these messages… sorry!
```suggestion
```

Reply:
Okay, I just found some messages containing them and some not. My favorite was to round the message up with the dot, but I can revert that.